compiler/verification/z3backend.ml

```diff
@@ -213,7 +213,7 @@ let translate_typ_lit (ctx : context) (t : typ_lit) : Sort.sort =
   | TBool -> Boolean.mk_sort ctx.ctx_z3
   | TUnit -> fst ctx.ctx_z3unit
   | TInt -> Arithmetic.Integer.mk_sort ctx.ctx_z3
-  | TRat -> failwith "[Z3 encoding] TRat type not supported"
+  | TRat -> Arithmetic.Real.mk_sort ctx.ctx_z3
```

Comment:
Are we sure we want to use Z3's real ? Didn't we agree we would use fixed-precision integers ?